.gitignore

```diff
@@ -15,3 +15,5 @@ docker/db
 tmp
 /docker/db
 build_rs_cov.profraw
+docker/monitoring/surveyor/jetstream
+docker/monitoring/surveyor/observations
```

Comment:
Are these post-ignored? (it seems they are still part of your changes). It's fine if we want to hardcode them locally but I'm just making sure that we avoid git tracking them if they are meant to be auto-generated. what do you think?

Reply:
They are not autogenerated. They need to be there for the surveryor to work.

Reply:
Then we shouldn't have them in the `.gitignore`, should incase we update their configurations (?) Sorry, I know I initially asked for it but it's now clear that they are regular Grafana dashboards

Reply:
I have removed these 2 lines from gitignore now. In these folders we have configs that we should keep. Sorry about the confusion. All good now.